dbt_copilot_helper/utils/application.py

```diff
@@ -87,6 +87,7 @@ def load_application(app: str = None, default_session: Session = None) -> Applic
     application.environments = {
         env["name"]: Environment(env["name"], env["accountID"], sessions)
         for env in [json.loads(p["Value"]) for p in response["Parameters"]]
+        if "name" in env
```

Comment:
Is this here to allow for the `/copilot/applications/{application}/environments/{environment}/addons` paths? Can we filter that out based on the parameter name in the line above, something along the lines of `[json.loads(p["Value"]) for p in response["Parameters"] if not p['Name'].endswith('addons')]`.

Reply:
Yeah thats the addons config for the environment, if someone adds a new one with a name property that doesn't describe an environment this will break. How about the following instead:

```python
{
    env["name"]: Environment(env["name"], env["accountID"], sessions)
    for env in [
        json.loads(p["Value"]) for p in response["Parameters"] 
        if len(p.replace(f"/copilot/applications/{application.name}/environments/", "").split('/')) == 1
    ]
}
```

This will mean if we add extra parameters namespaced to environments that do have a name prop, we're not suddenly failing here.

Reply:
Do you mean check that the parameter name is in the correct format? i.e.:
```
if len(p['Name'].replace(f"/copilot/applications/{application.name}/environments/", "").split('/')) == 1
```

Reply:
Yeah checking the parameter name rather than checking the parameter value has a name field.